examples/federated-learning/tff/nvflare/terraform/iam.tf

```diff
@@ -0,0 +1,25 @@
+# Copyright 2023 Google LLC
```

Comment:
Refactor this Terraform stuff to be a module inside the `/terraform` directory.

Then, in `terraform/main.tf`, add an invocation to this module as we did with the cross-device one.

Reply:
Done